probe-rs/src/debug/debug_info.rs

```diff
@@ -1562,28 +1562,73 @@ fn add_to_address(address: u64, offset: i64, address_size_in_bytes: usize) -> u6
 
 #[cfg(test)]
 mod test {
-    use std::path::{Path, PathBuf};
+    use std::{
+        fs,
+        path::{Path, PathBuf},
+    };
 
     use crate::{
         architecture::arm::core::{
             exception_handling::{ArmV6MExceptionHandler, ArmV7MExceptionHandler},
             registers::cortex_m::CORTEX_M_CORE_REGISTERS,
         },
         core::exception_handler_for_core,
-        debug::{DebugInfo, DebugRegister, DebugRegisters},
+        debug::{DebugInfo, DebugRegister, DebugRegisters, Variable, VariableCache},
         test::MockMemory,
         CoreDump, RegisterValue,
     };
 
-    fn debug_info(filename: &str) -> DebugInfo {
-        let path = Path::new(filename);
-
-        let mut base_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
-        base_dir.push("tests");
+    /// Helper function to get the path to a file in the `tests` directory.
```

Comment:
I would prefer if things did not say `Helper function` everywhere. Docs should describe what it does plain and simple.

Reply:
Fair comment. I sometimes document as a stream of (un)consciousness ... will update it.

Reply:
Fixed.